yarn.lock

```diff
@@ -7055,11 +7055,16 @@ typed-styles@^0.0.7:
   resolved "https://registry.yarnpkg.com/typed-styles/-/typed-styles-0.0.7.tgz#93392a008794c4595119ff62dde6809dbc40a3d9"
   integrity sha512-pzP0PWoZUhsECYjABgCGQlRGL1n7tOHsgwYv3oIiEpJwGhFTuty/YNeduxQYzXXa3Ge5BdT6sHYIQYpl4uJ+5Q==
 
-typescript@^4.9, typescript@~4.9.5:
+typescript@^4.9:
```

Comment:
I am prepping an update to our shared eslint config and dev-utils packages to update to TS 5 also, so we don't still have TS 4 in the mix!

Reply:
Updated @xh/hoist-dev-utils to newly built v7-SNAP, includes transitive dep on updated @xh/eslint-config v6 with TS 5.x support.